<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OSMDownloadProjectWidgetForm</class>
 <widget class="QWidget" name="OSMDownloadProjectWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListView" name="projectsListView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="createProject">
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="updateProject">
         <property name="text">
          <string>Update</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="loadProject">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteProject">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
